teleop_twist_keyboard.py

```diff
@@ -33,100 +66,113 @@
 """
 
 moveBindings = {
-		'i':(1,0,0,0),
-		'o':(1,0,0,-1),
-		'j':(0,0,0,1),
-		'l':(0,0,0,-1),
-		'u':(1,0,0,1),
-		',':(-1,0,0,0),
-		'.':(-1,0,0,1),
-		'm':(-1,0,0,-1),
-		'O':(1,-1,0,0),
-		'I':(1,0,0,0),
-		'J':(0,1,0,0),
-		'L':(0,-1,0,0),
-		'U':(1,1,0,0),
-		'<':(-1,0,0,0),
-		'>':(-1,-1,0,0),
-		'M':(-1,1,0,0),
-		't':(0,0,1,0),
-		'b':(0,0,-1,0),
-	       }
-
-speedBindings={
-		'q':(1.1,1.1),
-		'z':(.9,.9),
-		'w':(1.1,1),
-		'x':(.9,1),
-		'e':(1,1.1),
-		'c':(1,.9),
-	      }
-
-def getKey():
-	tty.setraw(sys.stdin.fileno())
-	select.select([sys.stdin], [], [], 0)
-	key = sys.stdin.read(1)
-	termios.tcsetattr(sys.stdin, termios.TCSADRAIN, settings)
-	return key
+    'i':(1,0,0,0),
+    'o':(1,0,0,-1),
+    'j':(0,0,0,1),
+    'l':(0,0,0,-1),
+    'u':(1,0,0,1),
+    ',':(-1,0,0,0),
+    '.':(-1,0,0,1),
+    'm':(-1,0,0,-1),
+    'O':(1,-1,0,0),
+    'I':(1,0,0,0),
+    'J':(0,1,0,0),
+    'L':(0,-1,0,0),
+    'U':(1,1,0,0),
+    '<':(-1,0,0,0),
+    '>':(-1,-1,0,0),
+    'M':(-1,1,0,0),
+    't':(0,0,1,0),
+    'b':(0,0,-1,0),
+}
+
+speedBindings = {
+    'q':(1.1,1.1),
+    'z':(.9,.9),
+    'w':(1.1,1),
+    'x':(.9,1),
+    'e':(1,1.1),
+    'c':(1,.9),
+}
+
+def getKey(settings):
+    tty.setraw(sys.stdin.fileno())
+    select.select([sys.stdin], [], [], 0)
+    key = sys.stdin.read(1)
+    termios.tcsetattr(sys.stdin, termios.TCSADRAIN, settings)
+    return key
 
 
 def vels(speed,turn):
-	return "currently:\tspeed %s\tturn %s " % (speed,turn)
+    return "currently:\tspeed %s\tturn %s " % (speed,turn)
+
+
+def main():
+    settings = termios.tcgetattr(sys.stdin)
+
+    rclpy.init()
+
+    node = rclpy.create_node('teleop_twist_keyboard')
+    pub = node.create_publisher(Twist, 'cmd_vel', qos=qos_profile_sensor_data)
```

Comment:
create_publishwer doesn't have a qos keyword but a `qos_profile` keyword (see https://github.com/ros2/rclpy/blob/master/rclpy/rclpy/node.py#L64), this argument is currently being ignored and the created publisher uses default qos profile